src/main/io/osd.c

```diff
@@ -460,6 +462,37 @@ static bool osdDrawSingleElement(uint8_t item)
     char buff[OSD_ELEMENT_BUFFER_LENGTH] = "";
 
     switch (item) {
+    case OSD_FLIP_ARROW: 
+        {
+            const int angleR = attitude.values.roll;
+            const int angleP = attitude.values.pitch; // still gotta update all angleR and angleP pointers.
+            if (IS_RC_MODE_ACTIVE(BOXFLIPOVERAFTERCRASH)) {
+                if (angleP > 0 && ((angleR > 175 && angleR < 180) || (angleR > -180 && angleR < -175))) {
+			        buff[0] = SYM_ARROW_SOUTH;
+			    }
+                else if (angleP > 0 && angleR > 0 && angleR < 175) {
+                    buff[0] = (SYM_ARROW_EAST + 2);
+                }
+                else if (angleP > 0 && angleR < 0 && angleR > -175) {
+                    buff[0] = (SYM_ARROW_WEST + 2);
+                }
+                else if (angleP <= 0 && ((angleR > 175 && angleR < 180) || (angleR > -180 && angleR < -175))) {
+                    buff[0] = SYM_ARROW_NORTH;
+                }
+                else if (angleP <= 0 && angleR > 0 && angleR < 175) {
+                    buff[0] = (SYM_ARROW_NORTH + 2);
+                }
+                else if (angleP <= 0 && angleR < 0 && angleR > -175) {
+                    buff[0] = (SYM_ARROW_SOUTH + 2);
+                }
+            }
+            else
```

Comment:
Please put `else` and the block beginnings / ends onto the same line, like `} else {`.